### DISCUSSION TITLE
Reaccessing black hole audio

### DISCUSSION THREAD

**Discussion by claireher** [ExistentialAudio/BlackHole#574] [Q&A]
If I recorded a video on my laptop using Photo Booth and accidentally had my internal speakers set to record through Black hole, is there a way for me to reaccess this audio and have it play via my Mac speaker? Basically, is audio recorded via Blackhole saved anywhere for me to access? I'm very new to the software.

**Comment by devinroth** [comment]
BlackHole doesn’t record any audio. Sorry.

• Devin
